### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by IlyaGrynyshyn**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -1,7 +1,28 @@
...
-    pass
+    Genre.objects.create(name="Western")
+    Genre.objects.create(name="Action")
+    Genre.objects.create(name="Dramma")
+    Actor.objects.create(first_name="George", last_name="Klooney")
+    Actor.objects.create(first_name="Kianu", last_name="Reaves")
+    Actor.objects.create(first_name="Scarlett", last_name="Keegan")
+    Actor.objects.create(first_name="Will", last_name="Smith")
+    Actor.objects.create(first_name="Jaden", last_name="Smith")
+    Actor.objects.create(first_name="Scarlett", last_name="Johansson")
```
Comment (pavlejviki) [inline main.py:16]: use a for loop to avoid code duplication

**Review by pavlejviki — CHANGES_REQUESTED** [review]

File: main.py [hunk 2]
```diff
@@ -1,7 +1,30 @@
 import init_django_orm  # noqa: F401
 
 from django.db.models import QuerySet
+from db.models import Actor, Genre
 
 
 def main() -> QuerySet:
-    pass
+    for genre in ["Western", "Action", "Dramma"]:
+        Genre.objects.create(name=genre)
```
Comment (viktoria-rybenchuk) [inline main.py:9]: add a blank line between two logical parts of the code, you can simply insert a blank line to visually separate the sections.

File: main.py [hunk 3]
```diff
@@ -1,7 +1,30 @@
...
+    for first_name, last_name in [
+        ["George", "Klooney"],
+        ["Kianu", "Reaves"],
+        ["Scarlett", "Keegan"],
+        ["Will", "Smith"],
+        ["Jaden", "Smith"],
+        ["Scarlett", "Johansson"]
+    ]:
```
Comment (viktoria-rybenchuk) [inline main.py:17]: To make the code more readable, you can save the list of first and last names in a variable and then use a for loop with that variable

File: main.py [hunk 4]
```diff
@@ -1,7 +1,30 @@
...
+    Actor.objects.filter(
+        first_name="George",
+        last_name="Klooney"
+    ).update(last_name="Clooney")
+    Actor.objects.filter(
+        first_name="Kianu",
+        last_name="Reaves"
+    ).update(first_name="Keanu", last_name="Reeves")
+    Genre.objects.filter(name="Action").delete()
+    Actor.objects.filter(first_name="Scarlett").delete()
```
Comment (viktoria-rybenchuk) [inline main.py:29]: Add a blank line to separate sections of your code for readability

**Review by viktoria-rybenchuk — CHANGES_REQUESTED** [review]

**Review by viktoria-rybenchuk — APPROVED** [review]
Good job!

**Pull request closed without merging**